bundles/org.openhab.ui/web/src/assets/definitions/blockly/blocks-notifications.js

```diff
@@ -69,8 +72,6 @@ export default function defineOHBlocks_Notifications (f7, isGraalJs) {
     init: function () {
       this.appendValueInput('message')
         .appendField('send notification')
-      this.appendValueInput('icon')
```

Comment:
Why remove the input for icon? 
I think icon is shown for log notifications in the Android app. so we can leave this deprecated block as it is.

Reply:
Probably done by mistake. Good catch.